pages/fundamentals/channel.html

```diff
@@ -51,9 +51,9 @@ <h3>Channel Introduction</h3>
 Along with transferring values (through channels),
 the ownership of some values may also be transferred between goroutines.
 When a goroutine sends a value to a channel,
-we can view the goroutine releases the ownership of some values.
+we can view the goroutine releases the ownership of the value.
 When a goroutine receives a value from a channel,
-we can view the goroutine acquires the ownerships of some values.
+we can view the goroutine acquires the ownership of the value.

```

Comment:
I'm sorry, the original descriptions are not very clear.
In fact, the ownerships of the values referenced by the transferred value are transferred.
The transferred value itself is copied in transferring.

Generally, a referencing relation is built with a pointer, but sometimes with index tables. 

I think it would be better to change them to:
```
When a goroutine sends a value to a channel,
we can view the goroutine releases the ownership of some values (which could be accessed through the sent value).
When a goroutine receives a value from a channel,
we can view the goroutine acquires the ownerships of some values (which could be accessed through the received value).
```

If you agree, could you change them as the above lines?

Reply:
Done ;)